src/bartiq/symbolics/ast_parser.py

```diff
@@ -129,23 +129,23 @@ def _replace_wildcards(expression):
 _WILDCARD_REPLACEMENT = _PreprocessingStage(matches=_contains_wildcard, preprocess=_replace_wildcards)
 
 
-def _contains_lambda(expression):
+def _contains_lambda(expression: str) -> bool:
```

Comment:
[comment] FYI, we don't require typing for private functions, but it's obviously always good to have it added, thanks!